### DISCUSSION TITLE
0.50.0 Missing windows build

### DISCUSSION THREAD

**Discussion by tylerhoot** [navidrome/navidrome#2649] [Q&A] [answered]
Just wondering the windows 64bit build is missing only one that i saw was for 0.49.3 and darwin_amd64.tar.gz doesnt work if that was the windows build

**Comment by deluan** [comment, marked as answer]
0.50.0 had a bug when running on Windows. This was fixed and 0.50.1 was just released.

**Comment by tylerhoot** [comment]
Thank you
